src/part_file.cpp

```diff
@@ -72,6 +72,9 @@ POSSIBILITY OF SUCH DAMAGE.
 
 namespace
 {
+	inline boost::int64_t calc_offset(boost::int64_t slot, int header_size, int piece_size)
```

Comment:
it almost seems warranted to make this a member, so it can pick up `m_header_size` and `m_piece_size` by itself.

Reply:
Don't you mind given name?

Reply:
`calc_offset()` seems fine, or maybe `slot_offset()`, unless it clashes with some existing names already